### ISSUE TITLE
tpm2_abrmd failing with "bad tcti conf string"

### ISSUE THREAD

**Issue by DomiNic5787** [tpm2-software/tpm2-abrmd#625]
Starting tpm2_abrmd is failing with `bad tcti conf string`, when no tcti is set.

Behavior does not fit to documentation:
```
If this option is omitted (or a NULL string provided) then a default TCTI is
used in it's default configuration.
```

Possible root cause: #619

**Comment by flihp** [comment]
Thanks @DomiNic5787, I think you're spot on here.

**Comment by flihp** [comment]
#626 is a candidate fix for this

**Comment by DomiNic5787** [comment]
Fix for this issue: #626
